docs/architecture/cloud-native/leverage-containers-orchestrators.md

```diff
@@ -19,7 +19,7 @@ Multi-tier architectures are nothing new, but with container-based applications
 ![Breaking up a monolithic app to use microservices in the backend.](./media/breaking-up-monolith-with-backend-microservices.png)
 **Figure 2-5**. Breaking up a monolithic app to use microservices in the backend.
 
-Once you start to have multiple containers that need to work together, it can be worthwhile to organize them at a higher level. By design containers only know about themselves. Organizing large numbers of containers and their shared dependencies, such as network configuration, is where orchestration tools come in to save the day! Kubernetes is a container orchestration platform designed to automate deployment, scaling, and management of containerized applications. It creates an abstraction layer on top of groups of containers, which it organized into what it calls *pods*. Pods run on worker machines referred to as *nodes*. The whole organized group is referred to as a *cluster*. Figure 2-6 shows the different components of a Kubernetes cluster.
+Once you start to have multiple containers that need to work together, it can be worthwhile to organize them at a higher level. By design containers only know about themselves. Organizing large numbers of containers and their shared dependencies, such as network configuration, is where orchestration tools come in to save the day! Kubernetes is a container orchestration platform designed to automate deployment, scaling, and management of containerized applications. It creates an abstraction layer on top of groups of containers, which it organizes into what it calls *pods*. Pods run on worker machines referred to as *nodes*. The whole organized group is referred to as a *cluster*. Figure 2-6 shows the different components of a Kubernetes cluster.
```

Comment:
which it organizes into what it calls *pods* -> this part of the sentence reads really strange since we don't generally have a subject after which. Who organizes that so I can try to provide a suggestion?

Reply:
All the "it" references in that sentence refer to Kubernetes.